### PR TITLE
Conventioning section titles

### LINKED ISSUE #2063: [GitHub Actions] Rename example section titles in "Workflow syntax for GitHub Actions"
### What article on docs.github.com is affected?

[Workflow syntax for GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions)

### What part(s) of the article would you like to see updated?

The example sections in this guide have adopted a convention of starting their titles with "Example [...]". e.g.
- ["Example including additional values into combinations"](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations)
- ["Example ignoring paths"](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths)
- ["Example running with more than one version of Node.js"](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-running-with-more-than-one-version-of-nodejs).

This convention sometimes makes it a bit challenging to construct a section title, since it's not beginning with a verb. 

To address this, we could adopt a new convention that renames all these section titles to begin with "Example: [verb]". e.g.:
Was: "Example including additional values into combinations"
Now: "Example:  Adding values to combinations"

Was: "Example ignoring paths"
Now: ""Example: Ignoring paths"

Was: "Example running with more than one version of Node.js"
Now: "Example: Running multiple versions of Node.js"

### PR DESCRIPTION
**Pull request by Perkles**
### Why:

Closes #2063 

### What's being changed:

 - All sections titles beginning with "Example: [verb]"

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by Perkles** [comment]
This article makes use of some reusables witch also match the pattern: "Example: [verb]" e.g.:

 - [reusables.github-actions.actions-on-examples](https://github.com/github/docs/blob/main/data/reusables/github-actions/actions-on-examples.md#example-using-a-single-event)
 - [reusables.actions.actions-group-concurrency](https://github.com/github/docs/blob/main/data/reusables/actions/actions-group-concurrency.md)

Should i apply the same convention for those titles as well?

**Comment by janiceilene** [comment]
@Perkles I think you should update the two reusables, as well! 

I searched for `data reusables.actions.actions-group-concurrency` in my editor and this article is the only place that reusable exists (it's used twice here). 

`data reusables.github-actions.actions-on-examples` is used in this article, as well as [Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#webhook-events). The reusable is the only place in that article where we use the "Example using..." phrasing, so replacing it with the new convention wouldn't make anything in the article look out of place.

Let me know when you've updated the reusables and are ready for a review ✨

**Review by fateking555 — DISMISSED** [review]

**Comment by Perkles** [comment]
Hello @janiceilene. 
Changes applied to reusables.
Ready to review ✨

**Comment by janiceilene** [comment]
Thanks @Perkles! I'll get this triaged ✨

File: content/actions/reference/workflow-syntax-for-github-actions.md [hunk 1]
```diff
@@ -723,7 +723,7 @@ jobs:
         uses: docker://gcr.io/cloud-builders/gradle
 ```
 
-#### Example using action inside a different private repository than the workflow
+#### Example: Using action inside a different private repository than the workflow
```
Comment (mchammer01) [inline content/actions/reference/workflow-syntax-for-github-actions.md:726]:
```suggestion
#### Example: Using an action inside a different private repository than the workflow
```

File: content/actions/reference/workflow-syntax-for-github-actions.md [hunk 2]
```diff
@@ -1054,7 +1054,7 @@ strategy:
 ```
 {% endraw %}
 
-#### Example including new combinations
+#### Example: Adding new combinations
```
Comment (mchammer01) [inline content/actions/reference/workflow-syntax-for-github-actions.md:1057]: Probably best to use the same name as the property in the workflow (`include`)
```suggestion
#### Example: Including new combinations
```

File: content/actions/reference/workflow-syntax-for-github-actions.md [hunk 3]
```diff
@@ -1119,7 +1119,7 @@ strategy:
 
 Prevents a workflow run from failing when a job fails. Set to `true` to allow a workflow run to pass when this job fails.
 
-#### Example preventing a specific failing matrix job from failing a workflow run
+#### Example: Preventing a failing matrix job from failing a workflow run
```
Comment (mchammer01) [inline content/actions/reference/workflow-syntax-for-github-actions.md:1122]:
```suggestion
#### Example: Preventing a specific failing matrix job from failing a workflow run
```

**Review by mchammer01 — DISMISSED** [review]
@Perkles - this LGTM ✨ 
I've made a few minor suggestions that I'm going to apply.

**Review by mchammer01 — APPROVED** [review]
Re-approving ✨

**Pull request merged**